ns1/rest/ipam.py

```diff
@@ -149,7 +149,7 @@ def search(self, network_id, prefix, callback=None, errback=None):
 class Networks(resource.BaseResource):
     ROOT = "ipam/network"
     INT_FIELDS = ["network_id"]
-    PASSTHRU_FIELDS = ["rt", "name", "desc", "kvps", "tags"]
+    PASSTHRU_FIELDS = ["rt", "name", "desc", "tags"]
```

Comment:
If `kvps` is being removed, should it also be removed from `PASSTHRU_FIELDS` on the `Address` class?